### PR TITLE
MongoDB tutorial code examples

### PR DESCRIPTION
**Pull request by lpozo**
**Where to put new files:**

- New files should go into a top-level subfolder, named after the article slug. For example: `my-awesome-article`

**How to merge your changes:** 

1. [Make sure the CI code style tests all pass (+ run the automatic code formatter if necessary).](https://github.com/realpython/materials/blob/master/README.md)
2. Find an RP Team member on Slack and ask them to review & approve your PR.
3. Once the PR has one positive ("approved") review, GitHub lets you merge the PR.
4. 🎉

### CODE REVIEW

File: python-mongodb/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Description
+
+The files in this folder provides the code for the examples discussed on: [How to Use Python to Interface With MongoDB: The Basics](https://realpython.com/blog/python/introduction-to-mongodb-and-python/) for more info.
```
Comment (bzaczynski) [inline python-mongodb/README.md:3]: ⚠️ **Typo**

```suggestion
The files in this folder provide code examples discussed in [How to Use Python to Interface With MongoDB: The Basics](https://realpython.com/blog/python/introduction-to-mongodb-and-python/).
```

Comment (bzaczynski) [reply python-mongodb/README.md:3]: @lpozo Please also remember to update this README file. Thanks!

**Comment by lpozo** [comment]
@bzaczynski I updated the README.md. I totally forgot about it! Thanks! Also, if the article's title changes, we'll need to update it again :)

**Review by bzaczynski — COMMENTED** [review]
@lpozo No worries :wink: This looks good to me!

**Comment by bzaczynski** [comment]
lgtm

**Review by jaschmitt — APPROVED** [review]

**Pull request merged**